### PR TITLE
Documentation Update: Fix outdated Helm chart instructions for prometheus. Add model loading example

### PR DESCRIPTION
**Pull request by GuanLuo**

### CODE REVIEW

**Review by mc-nv — APPROVED** [review]

**Review by dyastremsky — APPROVED** [review]

File: deploy/aws/README.md [hunk 1]
```diff
@@ -126,13 +126,13 @@ by Grafana. The inference server helm chart assumes that Prometheus
...
 
-Use the prometheus-operator to install these components. The
+Use the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) to install these components. The
 *serviceMonitorSelectorNilUsesHelmValues* flag is needed so that
 Prometheus can find the inference server metrics in the *example*
 release deployed below.
 
 ```
-$ helm install --name example-metrics --set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false stable/prometheus-operator
+$ helm install example-metrics --set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false prometheus-community/kube-prometheus-stack
```
Comment (rmccorm4) [inline deploy/aws/README.md:135]: Can you replace all instances of the following for consistency in using `helm3`?:
- `helm install --name <name>` -> `helm install <name>` 
- `helm delete --purge <name>` -> `helm uninstall <name>`

If you check your `history` from testing this, I believe these are the versions of the commands you should've used. Use of `--name` in [helm install](https://helm.sh/docs/helm/helm_install/) and [helm delete](https://helm.sh/docs/helm/helm_delete/) was removed in `helm3`.

Comment (GuanLuo) [reply deploy/aws/README.md:135]: Will update. FYI, seems like in helm3, `delete` is marked as alias of `uninstall`, so it is still available but the usage is following `uninstall`

File: docs/protocol/extension_model_repository.md [hunk 2]
```diff
@@ -163,6 +163,38 @@ $repository_load_error_response =
...
+      ],
+      "outputs": [{
+          "name": "OUTPUT0",
+          "datatype": "FP32",
+          "shape": [ 1 ]
+        }
+      ]
+    },
+
+    "file:1/model.onnx" : ""<base64-encoded-file-content>"
```
Comment (rmccorm4) [inline docs/protocol/extension_model_repository.md]: is this an extra quote?
```suggestion
    "file:1/model.onnx" : "<base64-encoded-file-content>"
```

nice example :+1:

File: deploy/aws/README.md [hunk 3]
```diff
@@ -243,8 +243,8 @@ NAME            REVISION  UPDATED                   STATUS    CHART
 example         1         Wed Feb 27 22:16:55 2019  DEPLOYED  triton-inference-server-1.0.0  1.0           default
 example-metrics	1       	Tue Jan 21 12:24:07 2020	DEPLOYED	prometheus-operator-6.18.0   	 0.32.0     	 default
 
-$ helm delete --purge example
-$ helm delete --purge example-metrics
+$ helm uninstall example
```
Comment (rmccorm4) [inline deploy/aws/README.md:246]: Can you also remove the `--name` arg from all the `helm install` commands? `--name` was removed in helm3 per previous comment

**Review by rmccorm4 — CHANGES_REQUESTED** [review]

**Pull request merged**